nodeup/pkg/model/sysctls.go

```diff
@@ -116,6 +116,13 @@ func (b *SysctlBuilder) Build(c *fi.ModelBuilderContext) error {
 			"")
 	}
 
+	if *b.Cluster.Spec.Docker.IPv6 {
+		sysctls = append(sysctls,
```

Comment:
We normally try to avoid this sort of magic, but I can see why it's important in this case.  But note that not all instances uses eth0.

I'm inclined to allow this though, because the risk of breakage is pretty low, because most people will not be specifying `ipv6` (i.e. this is opt-in)